pkg/storage/metrics.go

```diff
@@ -155,6 +155,9 @@ var (
 	metaSysCount = metric.Metadata{
 		Name: "syscount",
 		Help: "Count of system KV pairs"}
+	metaWritesPerSecond = metric.Metadata{
+		Name: "writes_per_second",
```

Comment:
I'm not sure that it should be. I mean, of course rates should typically be handled by the UI/prometheus, but the goal in this case is to graph the data used by the rebalancing logic, which won't be identical to what we'd get by maintaining a counter metric and doing the rate later. The rebalancing logic is getting something approximating the average over the last 20-25 minutes.